packages/vite-plugin-svelte/src/index.ts

```diff
@@ -104,25 +101,6 @@ export function svelte(inlineOptions?: Partial<Options>): Plugin {
 				return importee; // query with svelte tag, an id we generated, no need for further analysis
 			}
 
-			if (ssr && importee === 'svelte') {
-				if (!resolvedSvelteSSR) {
-					resolvedSvelteSSR = this.resolve('svelte/ssr', undefined, { skipSelf: true }).then(
```

Comment:
Since Vite can't resolve `svelte` to the `node` exports, I wonder if we should revert this file change, and swap out `svelte/ssr` to `svelte`, so that we "help" Vite out to resolve it. IIRC `this.resolve`, which uses the `resolver` plugin has support for exports map, so it should work. But I could completely wrong.

If so, I think we should left using `svelte/ssr` at the meantime

Reply:
@benmccann Nevermind, tested this and it doesn't work